frontend/components/commons/sidebar/SidebarMenu.vue

```diff
@@ -35,15 +35,28 @@
         <a class="sidebar__info__button"
           href="#"
           data-title="Annotate"
-          @click="$emit('onEnableAnnotationView', true)"
+          @click="$emit('onChangeViewMode', 'annotate')"
         >
           <svgicon
-            v-show="annotationMode"
+            v-show="currentViewMode === 'annotate'"
             class="sidebar__info__icon-help"
             name="check3"
           ></svgicon>
           <svgicon name="annotate-view"></svgicon>
         </a>
+        <a class="sidebar__info__button"
+          v-if="showDefineRules"
+          href="#"
+          data-title="Define rules"
+          @click="$emit('onChangeViewMode', 'labelling-rules')"
+        >
+          <svgicon
+            v-show="currentViewMode === 'labelling-rules'"
+            class="sidebar__info__icon-help"
+            name="check3"
+          ></svgicon>
+          <svgicon name="labelling-rules-view"></svgicon>
+        </a>
```

Comment:
We can encapsulate this as a `SidebarButton` component instead of repeat every time the same code.